bin/compile

```diff
@@ -57,14 +57,9 @@ case "$channel" in
     ;;
 esac
 
-stack=${STACK:-heroku-16}
```

Comment:
This default is:
(a) not needed (since `STACK` is always set in the build system),
(b) undesirable even if (a) wasn't the case, since the buildpack shouldn't install a random stacks packages if the stack wasn't set.